### PR TITLE
fix(user): allow users to choose notification email. Fixes MA-134

### PR DESCRIPTION
**Pull request by LeonVreling**
Add the option for users to select which email address they want to use for notifications. Fixes [MA-134](https://myaegee.atlassian.net/browse/MA-134)

### CODE REVIEW

File: src/views/core/members/EditPrimaryEmailModal.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="modal-card">
+    <header class="modal-card-head">
+      <p class="modal-card-title">Set primary email</p>
+      <button class="delete" aria-label="close" @click="$parent.close()"></button>
+    </header>
+
+    <section class="modal-card-body">
+      <div class="field">
+        <label class="label">Primary email</label>
```
Comment (WikiRik) [inline src/views/core/members/EditPrimaryEmailModal.vue:10]: Maybe add a tooltip explaining what the primary email means?

File: src/views/core/members/Single.vue [hunk 2]
```diff
@@ -38,6 +38,13 @@
             </button>
           </div>
 
+          <div class="field is-grouped" v-if="can.edit">
```
Comment (WikiRik) [inline src/views/core/members/Single.vue:41]: For what we discussed today; if you add to this that you can only see it when gsuite_id is not empty then we can merge this soon

File: src/views/core/members/EditPrimaryEmailModal.vue (hunk 1 above)
Comment (WikiRik) [reply src/views/core/members/EditPrimaryEmailModal.vue:10]: This would still be nice btw

**Comment by LeonVreling** [comment]
Updated both the requests @WikiRik, sorry for the delay 😅

**Review by WikiRik — APPROVED** [review]

**Pull request merged**